model/config.go

```diff
@@ -489,6 +491,21 @@ func (o *Config) SetDefaults() {
 		*o.SqlSettings.QueryTimeout = 30
 	}
 
+	if o.FileSettings.EnableClientSideEncryption == nil {
+		o.FileSettings.EnableClientSideEncryption = new(bool)
+		*o.FileSettings.EnableClientSideEncryption = false
+	}
+
+	if *o.FileSettings.EnableClientSideEncryption {
+		if o.FileSettings.ClientSideEncryptionKey == nil || *o.FileSettings.ClientSideEncryptionKey == "" {
+			o.FileSettings.ClientSideEncryptionKey = new(string)
+			*o.FileSettings.ClientSideEncryptionKey = NewRandomString(32)
+		}
+	} else {
+		o.FileSettings.ClientSideEncryptionKey = new(string)
```

Comment:
Do we want to clear the key if encryption is disabled? I think we usually just leave it as is as long as its not configured. Maybe change the `else` part to `else if o.FileSettings.ClientSideEncryptionKey == nil`